libs/application/templates/children-residence-change/src/fields/Overview/fileSignatureReducer.tsx

```diff
@@ -21,11 +21,19 @@ export enum FileSignatureActionTypes {
 
 type Action =
   | { type: Exclude<FileSignatureActionTypes, FileSignatureActionTypes.ERROR> }
-  | { type: FileSignatureActionTypes.ERROR; error: string; status: ErrorStatus }
+  | {
+      type: FileSignatureActionTypes.ERROR
+      error?: number
```

Comment:
Should this be optional? Just thinking if we should pass a default error code if there isn't one